.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,97 @@
+
+name: "Docker publish"
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      generic_tag:
+        required: true
+        type: string        
+
+jobs:
```

Comment:
This workflow, on the contrary, might need packages:write for ghcr publish?

Reply:
Or permissions: inherit?

Reply:
That would be really nice to also have inheritance for permissions but I didn't find anything like this. 
However, I added 
```
permissions: 
  packages: write
```
in f2a65045

and also align those permissions in `main.yml` and `release.yml` workflows

Reply:
Thanks much! 
> I didn't find anything like this in documentation.

I must've mixed this with secrets :) 

I'm currently waiting to get the secrets configured, we'll merge once I get that done